rose/server/game.py

```diff
@@ -9,6 +9,8 @@
 from . import player
 from . import score
 
+FORMAT = '%(asctime)-15s %(levelname)s [%(name)-8s] %(message)s (%(funcName)s:%(lineno)d)'
```

Comment:
Some issues in the format:
- We don't need to set the timestamp width or change the default alignment
- The levelname width differ, we like to align it so the next fields are aligned
  the widest level name is "WARNING", so we need -%7s
- We don't need to align the logger name
- In the issue I described (func:lineno), but I meant module:lineno, sorry for the confusion
- We use the same format in rose-server, rose-client, and rose-admin - so we want to keep one copy of the format, best in rose/common/config.py